carbon/app.py

```diff
@@ -98,20 +97,192 @@
     "PART-TIME FLEXIBLE/LL",
 )
 
-ENV_VARS = (
-    "FTP_USER",
-    "FTP_PASS",
-    "FTP_PATH",
-    "FTP_HOST",
-    "FTP_PORT",
-    "CARBON_DB",
-)
+
+def _add_article(xml_file: IO, article: dict[str, Any]) -> None:
+    """Create an XML element representing an article and write it to an output file.
```

Comment:
These new docstrings are really great and make the app much more friendly to newcomers